content/en/docs/architecture/step-registry.md

```diff
@@ -409,86 +427,81 @@ ref:
     limits:
       memory: 4Gi
   env:
-  - name: OPENSHIFT_TEST_SUITE
+  - name: TEST_SUITE
 {{< / highlight >}}
 
-A test or workflow that utilizes this step must give a value to the `OPENSHIFT_TEST_SUITE` parameter, which will be available as an environment
-variable when it is executed. Different tests can be generated by setting different values, which can make generating simple
-variations easier. More complex combinations are encouraged to use separate steps instead.
+Omitting a default value makes `TEST_SUITE` a required parameter. A test that
+wishes to use this step must give the parameter a value in its corresponding
+`env` section. Failing to do so will result in a validation error.
 
-Each parameter declaration in the `env` section consists of the following fields:
-
-* `name`: environment variable name
-* `default` (optional): the value assigned if no other node in the hierarchy
-  provides one (described in the [hierarchical propagation](#hierarchical-propagation)
-  section).
-* `documentation` (optional): a textual description of the parameter
+{{< highlight yaml >}}
+tests:
+- as: e2e
+  steps:
+    test:
+    - ref: openshift-e2e-test
+    env:
+      TEST_SUITE: openshift/conformance/parallel
+{{< / highlight >}}
```

Comment:
It is unclear what this snippet purpose is. It immediately follows the "Failing to do so will result in a validation error" sentence so I expected it to show the "wrong" configuration, but it is actually correct. So I think the snippet needs a lead paragraph that says what it shows.